htdocs/societe/class/societe.class.php

```diff
@@ -1550,7 +1545,7 @@ public function update($id, $user = '', $call_trigger = 1, $allowmodcodeclient =
 				$sql .= ",localtax2_value =0.000";
 			}
 
-			$sql .= ",capital = ".($this->capital == '' ? "null" : $this->capital);
+			$sql .= ",capital = ".($this->capital == 0 ? "null" : $this->capital);
```

Comment:
If 0, we must store 0, if null, we must store "null".

Reply:
Ok, you are right 👍